components/inputs/input.vue

```diff
@@ -415,6 +422,11 @@ export default Vue.extend({
       // Make sure value is clamped to max if it exists.
       let value: String | Number = target.value
 
+      // use key to refresh input field to eliminate potential trailing decimal point
+      if (value.trim() !== '' && !value.includes('.')) {
+        this.resetInputKey++
```

Comment:
Let's try to find a better way to handle this. Maybe a force re-render of the component?